app/services/clean_unreceived_matches.rb

```diff
@@ -0,0 +1,17 @@
+class CleanUnreceivedMatches
+  def perform
+    ActiveRecord::Base.transaction do
+      Match.missing.each do |m|
+        UnreceivedMatch.create(
```

Comment:
Use `create!` (notice the `!`) or this will not throw exceptions.

Reply:
:+1: